### DISCUSSION TITLE
Move project from a server to another or to cloud and viceversa and backup it.

### DISCUSSION THREAD

**Discussion by mgesmundo** [directus/directus#15077] [Archive]
Hi there,
this is an awesome project! I discovered it and I love it!
There is a way to move a whole project from a local installation to another local server or the cloud server and viceversa (with all custom modules, displays, etc.)?
There is a way to backup locally a cloud project?
Thank you in advance.
All the best.
Marcello

**Comment by rijkvanzanten** [comment]
> There is a way to move a whole project from a local installation to another local server or the cloud server and viceversa (with all custom modules, displays, etc.)?

All configuration is stored in the database, so as long as you backup the database, the uploads folder (assuming you use the local storage adapter), and the extensions folder, you should have everything 👍🏻 

> There is a way to backup locally a cloud project?

If you're talking about Directus Cloud specifically, you can reach out through the chat widget on [the Cloud Dashboard](https://directus.cloud) 🙂

**Reply by mgesmundo** [reply]
Hi @rijkvanzanten when a project is not in production is really easy, but when you have an already deployed project and you want to update it in a dev enviroment, how to mode the new settings (collections, endpoints, etc.) into the production environment?
The backup/restore is not possible.
Thank you.

**Reply by rijkvanzanten** [reply]
> The backup/restore is not possible.

Coming very very soon 🙂  https://github.com/directus/directus/pull/14713

**Reply by erikdemarco** [reply]
> > The backup/restore is not possible.
> 
> Coming very very soon 🙂 #14713

It's 2026. Still doesn't have backup restore for self hosted version? I mean not sql dump. But all the data in instance (file storage, workflow, configurations, etc)

**Reply by rijkvanzanten** [reply]
> I mean not sql dump. But all the data in instance

All the data in the instance is _in_ the sql database. If you want to do a full backup / restore like a point-in-time backup, by far the easiest way to achieve that is to indeed make a sql dump and copy of your file system. A lot of cloud providers provide native point-in-time backups for database servers and versioned buckets now, making whatever we could cook up as a "directus dump" inferior straight away
